DQMOffline/Trigger/python/TopMonitoring_cff.py

```diff
@@ -1,4 +1,149 @@
 import FWCore.ParameterSet.Config as cms
 
+from DQMOffline.Trigger.TopMonitor_cfi import hltTOPmonitoring
+
+eleJet_jet = hltTOPmonitoring.clone()
+eleJet_jet.FolderName = cms.string('HLT/TopHLTOffline/TopMonitor/EleJet/JetMonitor')
+eleJet_jet.nmuons = cms.uint32(0)
+eleJet_jet.nelectrons = cms.uint32(1)
+eleJet_jet.njets = cms.uint32(2)
+eleJet_jet.eleSelection = cms.string('pt>50 & abs(eta)<2.1')
+eleJet_jet.jetSelection = cms.string('pt>30 & abs(eta)<2.4')
+eleJet_jet.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_Ele30_eta2p1_WPTight_Gsf_CentralPFJet35_EleCleaned_v*')
+eleJet_jet.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_Ele30_eta2p1_WPTight_Gsf_v*',
+                                                             'HLT_Ele35_WPTight_Gsf_v*',
+                                                             'HLT_Ele38_WPTight_Gsf_v*',
```

Comment:
35 GeV was an oversight and will be added to EGM tag-n-probe. Thanks for the spot